### ISSUE TITLE
[Feature Request] End cards

### ISSUE THREAD

**Issue by f-o** [yuliskov/SmartTube#343]
This is a feature request to add End Cards to the end of video playback.
This feature should be toggle able (on/off) either from settings or from the player itself.

**Comment by jnkstry** [comment]
+1

**Comment by f-o** [comment]
I know you've been employed to help clean up the massive amount of open issues, and I think it's great that we finally get a chance to slim down the backlog.

But please, with all due respect @ACR-Jeff, do spend a bit more time on your work.

You've clearly closed this issue by mistake, as End Cards are not yet supported, and your explanation has nothing to do with the issue stated here.
Furthermore, there are multiple duplicate entries for this exact request, and I do not see any of those being prior to this issue from 2021, as I did do a search myself back then.
You should mark the other ones as duplicates, and link them here instead. This has been the norm on Github for many years.

If you're offering your help to benefit the project, I have nothing but respect for you.
But if you're doing it to just close down valid issues, I think you should reconsider your kindness.

**Comment by ACR-Jeff** [comment]
@f-o You're correct, I stand corrected, Thank you, I had to do some searching on what "End Cards" are since they are new to me, I now understand clearer, With your message it gave me the option to do some learning on this matter,
 
Respectfully you should also take into account that this is 5 years old without updates through 3k issues, It could have been closed as stale without explanation (On most repos it most likely would have been), But in all fairness I choose to give an explanation even though it was incorrect, We can only get to one at a time in all fairness, 1 of us would have come across duplicates and then marked them as duplicates to yours as it is the elder,
Please understand and please be patient with us. But I am also thankful that you've come forward, With more of an explanation in the OP would have helped with the circumstance.
